edu_edfi_airflow/providers/edfi/transfers/edfi_to_s3.py

```diff
@@ -271,7 +305,14 @@ def pull_edfi_to_object_storage(self,
 
             # Connect to object storage and copy file
             tmp_file.seek(0)  # Go back to the start of the file before copying to object storage.
-            object_storage.upload_from(tmp_file, force_overwrite_to_cloud=True)
+            with object_storage.open("wb") as storage_file:
+                storage_file.write(tmp_file.read())
+        
+        # Clean up the temporary file
+        try:
```

Comment:
Is this cleanup step something I missed in my original implementation, or is it a side effect of using a named temporary file? If the latter, why not use an unnamed version?